### ISSUE TITLE
Using mountType: 9p in override.yaml causes rancher desktop to fail on starting k3s

### ISSUE THREAD

**Issue by chancez** [rancher-sandbox/rancher-desktop#2144]
### Actual Behavior

I added my `override.yaml` because I forgot to add it before starting rancher desktop.

```
cp scratch/override.yaml ~/Library/Application\ Support/rancher-desktop/lima/_config/override.yaml       
```

Then I clicked on "Reset Kubernetes", which I can't recall, if it actually recreates the VM or not, but then it failed with errors after the reset.

After this occurs, the only way to resolve it is to factory reset.

### Steps to Reproduce

- Start RD
- Add `override.yaml` to RD lima config directory.
- Click "Reset Kubernetes" in RD
- See error

### Result

```
Kubernetes Error
Rancher Desktop 1.3.0 - darwin (x64)
Error Starting Kubernetes
Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1
Last command run:
/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl shell --workdir=. 0 sudo bin/install-k3s v1.22.7+k3s1 /Users/chancezibolski/Library/Caches/rancher-desktop/k3s

Context:
Installing k3s

Some recent logfile lines:
 * /var/log/containerd.log: creating file
 * Starting Container Daemon ... [ ok ]
Directory /Users/chancezibolski/Library/Caches/rancher-desktop/k3s/v1.22.7+k3s1 does not exist
2022-04-29T17:36:38.300Z: + limactl shell --workdir=. 0 sudo bin/install-k3s v1.22.7+k3s1 /Users/chancezibolski/Library/Caches/rancher-desktop/k3s
2022-04-29T17:36:38.301Z: Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1
2022-04-29T17:36:38.305Z: Error starting lima: Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1
    at ChildProcess.<anonymous> (/Applications/Rancher Desktop.app/Contents/Resources/app.asar/dist/app/background.js:17:141690)
    at ChildProcess.emit (node:events:390:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
```

### Expected Behavior

I wanted to have it recreate the VM with my override settings.

### Additional Information

override.yaml:
```
env:
  K3S_EXEC: '--kube-apiserver-arg=feature-gates=EphemeralContainers=true'
provision:
  - mode: system
    script: |
      set -e
      apk add \
        bash \
        bind-tools \
        curl \
        findmnt \
        tmux \
        util-linux
  - mode: system
    script: |
      #!/bin/sh
      set -e

      # needed for cilium
      mount bpffs -t bpf /sys/fs/bpf
      mount --make-shared /sys/fs/bpf
      mount --make-shared /sys/fs/cgroup
      mount --make-shared /sys/fs/cgroup/unified
hostResolver:
  enabled: false
  # ipv6: true

disk: 150GiB

mountType: 9p
```

### Rancher Desktop Version

1.3.0

### Rancher Desktop K8s Version

1.27.7

### Which container runtime are you using?

containerd (nerdctl)

### What operating system are you using?

macOS

### Operating System / Build Version

Darwin dune.local 21.4.0 Darwin Kernel Version 21.4.0: Fri Mar 18 00:46:32 PDT 2022; root:xnu-8020.101.4~15/RELEASE_ARM64_T6000 arm64

### What CPU architecture are you using?

arm64 (Apple Silicon)

**Comment by chancez** [comment]
The original issue stated this happens in reset, but it's actually occurring if I do these steps with a fresh RD setup and copy the override.yaml to the correct location before starting rancher-desktop.

**Comment by chancez** [comment]
Also sometimes factory reset seems to "not work" in this situation after the error occurs. I see a blue loading bar very briefly, and the UI never exits.

**Comment by chancez** [comment]
It seems this might be due to the 9p mountType specifically.

**Comment by jandubois** [comment]
This could be a duplicate of #2134, but I'm still waiting for confirmation that that issue was also trying `9p`.

I can reproduce the problem with `9p` when starting via RD, even though it works fine for me with plain lima. I will investigate this on Monday.
